packages/admin-ui-docs/content/table/code.mdx

```diff
@@ -383,7 +417,113 @@ function Example() {
 render(<Example />)
 ```
 
-##### Currency
+| Name   | Type                                                      | Description              | Required |
+| ------ | --------------------------------------------------------- | ------------------------ | -------- |
+| type   | `plain`                                                   | Plain resolver type      | ✅       |
+| render | `(props: ResolverRenderProps<ReactNode, T>) => ReactNode` | Resolver render function | 🚫       |
+
+</Section>
+
+<Section hasExplanation>
+
+### Text
+
+The text resolver should be mainly used to render a text and a optional description just below it. It must be used for cells belonging to the name column
```

Comment:
```suggestion
The text resolver should be mainly used to render a text and an optional description just below it. It must be used for cells belonging to the name column
```

Reply:
Don't understand what you mean by "cells belonging to the name column"

Reply:
Ah a column with type name? I think "belonging to a 'name' column" would be better if that's it